group.py

```diff
@@ -2,4 +2,49 @@
 
 # Your code to go here...
 
-my_group =
+# define individual people
+
+liam = {'Name': 'Liam',
+        'Age': 22,
+        'Job': 'Student (MRes Medical Imaging)',
+        'Connections': [['Ed', 'Colleague'],
+                        ['Premal', 'Colleague'],
+                        ['Wang', 'Colleague']]}
+
+premal = {'Name': 'Premal',
+        'Age': 21,
+        'Job': 'Student (MSc Astrophysics)',
+        'Connections': [['Ed', 'Colleague'],
+                        ['Liam', 'Colleague'],
+                        ['Wang', 'Colleague']]}
+
+ed = {'Name': 'Ed',
+        'Age': 22,
+        'Job': 'Student (MSc Scientific Computing)',
+        'Connections': [['Liam', 'Colleague'],
+                        ['Premal', 'Colleague'],
+                        ['Wang', 'Colleague']]}
+
+wang = {'Name': 'Wang',
+        'Age': 32,
+        'Job': 'Student (MSc Physics and Engineering in Medicine)',
+        'Connections': [['Liam', 'Colleague'],
+                        ['Premal', 'Colleague'],
+                        ['Ed', 'Colleague']]}
+
+morgan = {'Name': 'Morgan',
+          'Age': 26,
+          'Job': [],
+          'Connections': []}
+
+my_group = [liam, premal, ed, wang]
+# morgan]
+
+print(my_group)
+
+for i in range(len(my_group)):
```

Comment:
It's really helpful that you included this code for debugging - makes it a lot easier to tell that the data structure is working as expected.

Reply:
I second this